dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -76,11 +76,23 @@ done
 # Set configuration common to both minishift and openshift
 # --------------------------------------------------------
 
+CHE_MULTI_USER=${CHE_MULTI_USER:-"false"}
 DEFAULT_COMMAND="deploy"
 COMMAND=${COMMAND:-${DEFAULT_COMMAND}}
-DEFAULT_CHE_IMAGE_REPO="docker.io/eclipse/che-server"
+
+if [ "$CHE_MULTI_USER" == "true" ]
```

Comment:
Use ${CHE_MULTI_USER} instead of $CHE_MULTI_USER to be more consistent with coding style in the rest of this script.

Reply:
And `then` keyword should be on the same line of the test for the same reason.

Reply:
The same applies to other if conditions you have added in the script